doc/src/devdocs/ast.md

```diff
@@ -59,7 +62,6 @@ call. Finally, chains of comparisons have their own special expression structure
 | `a==b`      | `(call == a b)`           |
 | `1<i<=n`    | `(comparison 1 < i <= n)` |
 | `a.b`       | `(. a (quote b))`         |
-| `a.(b)`     | `(. a b)`                 |
```

Comment:
I removed `a.(b)` in "Operators" table (here) and mention it as `f.(x)` in "Calls" table since it seems the documentation does not match with the implementation:

```scm
> (julia-parse "a.(b)")
(|.| a (tuple b))
```